primer/src/Primer/Def/Utils.hs

```diff
@@ -40,12 +40,6 @@ nextIDTypeDef (TypeDefPrim (PrimTypeDef ps _)) =
   succ $ foldlOf' (folded % _2 % kindIDs) max minBound ps
 {-# INLINE nextIDTypeDef #-}
 
-globalInUse :: Foldable f => GVarName -> f Def -> Bool
-globalInUse v =
-  anyOf
-    (folded % #_DefAST % #astDefExpr % to freeGlobalVars)
-    (Set.member v)
-
```

Comment:
Alternatively, we could mark this as a Weeder root.

Reply:
I don't think there is much point in keeping this around. It can always be fished out of the repo's history if needed.